### PR TITLE
Fix number format over a million

### PR DESCRIPTION
**Pull request by rapahaeru**
👍  @diegocesar 

@erikatakahara

### CODE REVIEW

**Comment by tisvasconcelos** [comment]
Que tal adicionar mais testes para pegar esses e outros casos?

@tisvasconcelos

**Comment by tisvasconcelos** [comment]
The toLocaleString() method couldn't help? 

new Number(10000000000).toLocaleString('pt-br') // 10.000.000.000
new Number(10000000000).toLocaleString('en-us') // 10,000,000,000

https://developer.mozilla.org/pt-
BR/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString
⚠️  It does not make the decimal places

File: test/formatTest.js [hunk 1]
```diff
@@ -17,5 +17,35 @@ define(['format'], function(format) {
 			var currency = format.currency(number, "R$");
 			assert.equal("R$ 10,50", currency);
 		});
+		it('should return currency for value over a thousand', function(){
+			var number = 1000.50;
+			var currency = format.currency(number, "R$");
+			assert.equal("R$ 1.000,50", currency);
+		});
+		it('should return currency for value over a thousand', function(){
```
Comment (diegocesar) [inline test/formatTest.js]: Shouldn't it be "over a million"?

@luiz @FernandaBernardo

File: test/formatTest.js [hunk 2]
```diff
@@ -17,5 +17,35 @@ define(['format'], function(format) {
...
+			var number = 1234567.89;
+			var currency = format.currency(number, "R$");
+			assert.equal("R$ 1.234.567,89", currency);
+		});
+		it('should return currency for string using dots instead of comma', function(){
+			var number = "10.20";
+			var currency = format.currency(number, "R$");
+			assert.equal("R$ 10,20", currency);
+		});
+		it('should return currency for string using comma instead of dots', function(){
```
Comment (diegocesar) [inline test/formatTest.js]: "instead of dots"?

@luiz @FernandaBernardo

**Pull request merged**